.gitignore

```diff
@@ -43,7 +43,6 @@ config.status
 configure
 libtool
 src/config/bitcoin-config.h
-src/config/bitcoin-config.h.in
```

Comment:
Thanks!

This change is already a part of https://github.com/hebasto/bitcoin/pull/166.